docs/src/components/navbar.tsx

```diff
@@ -0,0 +1,83 @@
+import { useMemo, useState } from "react";
+
+import { buttonVariants } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { ScrollArea } from "@/components/ui/scroll-area";
+import { FunctionTag } from "@/components/function-tag";
+import { cn } from "@/lib/utils";
+
+import { toPairs } from "../../../src";
+
+export const Navbar = ({
+  entries,
+  onSelect,
+}: {
+  entries: Record<string, Array<{ name: string; tags: Array<string> }>>;
+  onSelect?: () => void;
+}) => {
```

Comment:
you don't need to take entries as a prop, you can import it directly in this component.
```suggestion
export const Navbar = ({
  onSelect,
}: {
  onSelect?: () => void;
}) => {
```